docs/semgrep-secrets/generic-secrets.md

```diff
@@ -0,0 +1,59 @@
+---
+slug: generic-secrets
+title: Scan for generic secrets
+hide_title: true
+description: Use Semgrep to identify generic secrets in your code.
+tags:
+ - Semgrep Assistant
+ - Semgrep Secrets
+---
+
+import PL from '@site/src/components/Placeholder';
+
+# Generic secrets (beta)
+
+:::info
+Generic secrets is in public beta. Please contact [Support](/support) to request access to this feature.
+:::
+
+Like Semgrep Secrets, which scans for specific secrets, **generic secrets detection** scans your code for the inadvertent inclusion of credentials, such as API keys, passwords, and access tokens using rules. However, generic secrets detection looks for common keywords, such as auth, key, or passwords, and flags anything nearby that appears to be a secret. It then analyzes the results to eliminate false positives, so you only see high-signal results likely to be true positives.
+
+## Prerequisites
+
+To scan your code for generic secrets, you must have the following:
+
+- Access to [Semgrep Secrets](/semgrep-secrets/getting-started).
+- [Semgrep Assistant](/semgrep-assistant/getting-started) enabled.
+- Semgrep CLI version `1.86.0` or higher running in your CI environment.
+
+Generic secrets does *not* work with local scans initiated by running the `semgrep ci` command.
+
+## Enable generic secrets
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep AppSec Platform](https://semgrep.dev/login).
+2. Go to **Settings > Deployment** and navigate to the **Secrets** section.
+3. Click the **Generic secrets** <i class="fa-solid fa-toggle-large-on"></i> toggle to turn on generic secrets.
+
+Once you have enabled generic secrets, your subsequent Semgrep Secrets scans automatically run with generic secrets rules. Semgrep also returns the following CLI message indicating that generic secrets detection has started: 
```

Comment:
This is not true, this only shows up when people have findings 

We are adding a note under the SECRET RULES part of the CLI that will show `AI augmented rules are active for secrets detection.` so people know the feature is active and running on scans.